From 6883062a080141ee18d86bc36bce4521d6403087 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 3 Nov 2023 08:12:38 +0000
Subject: IWYU: add functional for std::invoke in WTF::VectorTypeOperations

Bug: 957519
Change-Id: I1f69e1330091054da06bc058561c8b97906959e0
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5001628
Reviewed-by: Kentaro Hara <haraken@chromium.org>
Auto-Submit: Stephan Hartmann <stha09@googlemail.com>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Reviewed-by: Michael Lippautz <mlippautz@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1219355}
---
 third_party/blink/renderer/platform/wtf/vector.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/blink/renderer/platform/wtf/vector.h b/third_party/blink/renderer/platform/wtf/vector.h
index 0f9fcf2a69915..fa950d7392a83 100644
--- a/third_party/blink/renderer/platform/wtf/vector.h
+++ b/third_party/blink/renderer/platform/wtf/vector.h
@@ -24,6 +24,7 @@
 #include <string.h>
 
 #include <algorithm>
+#include <functional>
 #include <initializer_list>
 #include <iterator>
 #include <type_traits>
-- 
2.39.2

